### PR TITLE
Implemented email feature after purchase

### PR DESCRIPTION
**Pull request by Chimexx**
- Added email feature once a successful purchase  is made

### CODE REVIEW

**Review by SunmadeBabafemi — APPROVED** [review]
should the payment confirmation email not contain the course the user is purchasing for proper records

**Comment by Chimexx** [comment]
> should the payment confirmation email not contain the course the user is purchasing for proper records

Yes, there are a lot of other things that will be included in the email, that will be discussed in the standup

File: package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
     "create-entity": "npm run typeorm entity:create -- -n "
   },
   "dependencies": {
-    "@nestjs-modules/mailer": "^1.5.1",
+    "@nestjs-modules/mailer": "^1.8.1",
```
Comment (PhilemonBrain) [inline package.json:30]: why did u update the version?

Comment (Chimexx) [reply package.json:30]: I was having issues with css, I decided to try another version and it worked.

Comment (PhilemonBrain) [reply package.json:30]: css??? care to explain?

Comment (Chimexx) [reply package.json:30]: The html file wasn't recognising css, either inline or external, so a post online was using using  v1.8.1, I decided to upgrade it, now it works. Before now css wasn't  working.

**Review by PhilemonBrain — APPROVED** [review]

**Pull request merged**